pkg/daemon/update.go

```diff
@@ -432,6 +432,26 @@ func (dn *Daemon) updateKernelArguments(oldConfig, newConfig *mcfgv1.MachineConf
 	return exec.Command("rpm-ostree", args...).Run()
 }
 
+// updateFIPS handles changes in FIPS
+func (dn *Daemon) updateFIPS(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+	if oldConfig.Spec.FIPS != newConfig.Spec.FIPS {
+		return nil
+	}
+	if dn.OperatingSystem != machineConfigDaemonOSRHCOS {
+		return fmt.Errorf("Updating FIPS on non-RHCOS nodes is not supported")
+	}
+
+	arg := "enable"
+	if !newConfig.Spec.FIPS {
+		arg = "disable"
+	}
+
+	cmd := "/usr/libexec/rhcos-tools/coreos-fips"
+	args := []string{arg}
+	dn.logSystem("Running %s %v", cmd, args)
+	return exec.Command(cmd, args...).Run()
```

Comment:
This looks like the right command to me. One thing to keep in mind is that, within OCP, the `coreos-fips` will not default to updating the kernel args and, instead, defer to MCO to own those args.

Reply:
Let's debate that in https://github.com/openshift/rhcos-tools/pull/2 since...it seems to me doing it that way would make everything unnecessarily more complex.

Reply:
I closed the PR which would offload kernel args to the MCO. This call should be good as is.